README.md

```diff
@@ -54,6 +54,113 @@ The following Visual Studio Code extensions are recommended:
 - [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
 - [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
 
+## Scripts and tasks
+
+Scripts for each `package.json` should follow these rules:
+
+### Nesting
+
+Some scripts may be nested under other scripts using the colon (`:`) symbol. This can interchangeably be used to mean either:
+
+- The nested script is a fragment of the parent script.
+  - E.g.:
+    ```json
+    {
+      "build": "npm run build:bundles && npm run build:definitions",
+      "build:bundles": "...",
+      "build:definitions": "..."
+    }
+    ```
+- The nested script is an alternative configuration of the parent script.
+  - E.g.:
+    ```json
+    {
+      "e2e": "cypress run --browser chrome",
+      "e2e:firefox": "cypress run --browser firefox",
+      "e2e:watch": "cypress open"
+    }
+    ```
+
+### `build` script
+
+- Generates all the files needed for a dependent project to run.
+- Doesn't expect another script from the same project to be run before this script.
+- If the build fails, exits with an error.
+
+### `dev` script
+
+- Generates all the files needed for a dependent project to run.
+- Doesn't expect another script from the same project to be run before this script.
+- Automatically re-runs when there's a code change in the project or its dependencies.
+- Doesn't exit until the user chooses to.
+- May serve the project with a URL.
+
+### `prod` script
```

Comment:
I'm not sure what name to use for this script 🤔 

I used `prod` because it contrasts with `dev`, but `prod` sounds like it could be a script meant to deploy to prod.

Reply:
Like dev, switching the script names might confuse the repo users, some may have local setups running commands for tests or that kind of thing.

Reply:
If I'm alone thinking this, ignore these comments 😄